### DISCUSSION TITLE
Question about buying plugin

### DISCUSSION THREAD

**Discussion by popbluesky** [wpappointments/wpappointments#222] [General]
After buy plugin,  Is the plugin available for download and installed?

**Comment by tomrobak** [comment]
> After buy plugin, Is the plugin available for download and installed?

That plugin doesn't exist yet. So nothing to install at the moment

**Reply by popbluesky** [reply]
thanks for your information

**Comment by wppoland** [comment]
For anyone finding this thread later: the free core plugin is not on WordPress.org yet, and lifetime license sales are paused until the project is complete. Work is active again and focused on shipping v1.0 of the free core plugin. Progress is tracked in this repo's issues and discussions.
